apistructs/issue.go

```diff
@@ -682,14 +682,16 @@ func (ipr *IssuePagingRequest) GetUserIDs() []string {
 // IssueExportExcelRequest 事件导出 excel 请求
 type IssueExportExcelRequest struct {
 	IssuePagingRequest
-	IsDownload bool `json:"isDownload"`
+	Locale     string `json:"locale"`
+	IsDownload bool   `json:"isDownload"`
 }
 
 // IssueImportExcelRequest 事件导入excel请求
 type IssueImportExcelRequest struct {
-	ProjectID uint64            `json:"projectID"`
-	OrgID     int64             `json:"orgID"`
-	Type      PropertyIssueType `json:"type"`
+	ProjectID   uint64            `json:"projectID"`
+	OrgID       int64             `json:"orgID"`
+	Type        PropertyIssueType `json:"type"`
```

Comment:
type should change to issueType

Reply:
this filed will be quote by others and front engineers use `propertyIssueType`, so keep PropertyIssueType
![image](https://user-images.githubusercontent.com/30427474/150118136-94e1adf4-09eb-4598-8444-7998a5450553.png)